### DISCUSSION TITLE
out

### DISCUSSION THREAD

**Discussion by ruochen800** [getlantern/lantern-forum-cn#792] [Q&A]
1. I created a new Windows account and finally got it authorized successfully, but when I switch back to the other account, it's no longer the Pro version. How can I fix this?

2. I previously sent you a payment receipt, where the amount due was 624, but the actual amount paid was 707. Since I can't see my username in Lantern, could you help me check which account was charged for the renewal? Is it kunpp82594120@163.com or ruochen800@gmail.com? Both email addresses received emails, and I can provide the eight-digit numbers from the subject lines: 75903839 and 75791581.

**Comment by MaxMKMax** [comment]
@ruochen800 Hi Dear Ruochen, thanks for reaching us again. We are happy to assist you on the matter or problem you facing.

1. For this case, you might need to have an authorization code for your device. Not sure if you can get the code? Please refer to this link for clearer information. If you are unable to get the code, feel free to reach me again and i will get the code from tech support to assist you on login on other devices.
https://lantern.io/zh-Hans/faq

2. (A) Sure i will help you to check which account for renewal. Will get back to you once the payment team is updating me.
2. (B) Regarding the overcharges issues was happened because of some technical issues from third parties exchange rate system. Regarding the additional charges, i would suggest we can add extra time for your pro account. I will get back to you after discussion with the team.

Thanks for being patiently wait us to solve the issues and appreciate you supporting Lantern. It's great to have you as our user.

Thank you so much and have a nice day!

cc: @greySkiesMarsh

**Reply by ruochen800** [reply]
Thank you for your reply.

I have already tried all the methods provided online, but none of them worked. Every time I try to authorize, I receive the same code, 910716, which indicates that it's an invalid authorization code. I've been using Lantern for four or five years, so I'm very familiar with the authorization process. 

For the past two years, I've been using this Windows 10 account. Although creating a new account allowed me to authorize successfully, I couldn't share it with the original account, and it's very difficult to use. Even though it shows that it's the Pro version, it functions just like the free version, with webpages barely opening and constant lagging.

If this issue can be resolved, please do so as soon as possible, or provide me with a new activation code. I've already wasted too much time on this. If it cannot be resolved, please don't give me false hope.

**Reply by MaxMKMax** [reply]
@ruochen800 Okay sure. Let me help you solve immediately. Sorry for the dissapointment and waiting. I will try my best to help you solve this issues as soon as possible. Thanks.

**Reply by ruochen800** [reply]
I haven't tried to authorize again, but my computer suddenly popped up and upgraded to the Professional version. I'm not sure why. So the authorization issue has been resolved, but there's still one remaining problem: the extra fee hasn't been credited with an extension yet. Please resolve this as soon as possible. Thank you!
